docs/src/userguide/language_basics.rst

```diff
@@ -44,72 +47,143 @@ the use of ‘early binding’ programming techniques.
 C variable and type definitions
 ===============================
 
-The :keyword:`cdef` statement is used to declare C variables, either local or
-module-level::
+C variables can be declared by 
 
-    cdef int i, j, k
-    cdef float f, g[42], *h
+* using the Cython language :keyword:`cdef` statement,
+* annotating variable by special cython type or
+* using function ``declare()``.
 
-and C :keyword:`struct`, :keyword:`union` or :keyword:`enum` types:
+Statement :keyword:`cdef` and function ``declare()`` can declare either local or
+module-level variables, but annotated version currently supports only local variables:
```

Comment:
```suggestion
The :keyword:`cdef` statement and ``declare()`` can define function-local and
module-level variables, but type annotations only affect local variables.
This is because type annotations are not Cython specific, so Cython keeps
the variables in the module dict (as Python values) instead of making them
module internal C variables. Use ``declare()`` in Python code to explicitly
define global C variables.
```